# DP: Recognize DW_LANG_Go in readelf (backport from trunk)

--- a/binutils/dwarf.c
+++ b/binutils/dwarf.c
@@ -1599,6 +1599,8 @@ read_and_display_attr_value (unsigned lo
 	case DW_LANG_D:			printf ("(D)"); break;
 	  /* DWARF 4 values.  */
 	case DW_LANG_Python:		printf ("(Python)"); break;
+	  /* DWARF 5 values.  */
+	case DW_LANG_Go:		printf ("(Go)"); break;
 	  /* MIPS extension.  */
 	case DW_LANG_Mips_Assembler:	printf ("(MIPS assembler)"); break;
 	  /* UPC extension.  */
